### PR TITLE
Create Discount & Category repositories (CRUD)

### PR DESCRIPTION
**Pull request by VolodymyrBushko**

### CODE REVIEW

File: src/main/java/com/exadel/discountwebapp/category/repository/CategoryRepository.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.exadel.discountwebapp.category.repository;
+
+import com.exadel.discountwebapp.category.entity.Category;
+import org.springframework.data.repository.CrudRepository;
+
+public interface CategoryRepository extends CrudRepository<Category, Long> {
```
Comment (ahlukhouski2) [inline src/main/java/com/exadel/discountwebapp/category/repository/CategoryRepository.java:8]: add @repository annotation

File: src/main/java/com/exadel/discountwebapp/discount/repository/DiscountRepository.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.exadel.discountwebapp.discount.repository;
+
+import com.exadel.discountwebapp.discount.entity.Discount;
+import org.springframework.data.repository.CrudRepository;
+
+public interface DiscountRepository extends CrudRepository<Discount, Long> {
```
Comment (ahlukhouski2) [inline src/main/java/com/exadel/discountwebapp/discount/repository/DiscountRepository.java:8]: add @repository annotation

File: src/main/java/com/exadel/discountwebapp/category/repository/CategoryRepository.java (hunk 1 above)
Comment (VolodymyrBushko) [reply src/main/java/com/exadel/discountwebapp/category/repository/CategoryRepository.java:8]: done

File: src/main/java/com/exadel/discountwebapp/discount/repository/DiscountRepository.java (hunk 2 above)
Comment (VolodymyrBushko) [reply src/main/java/com/exadel/discountwebapp/discount/repository/DiscountRepository.java:8]: done

**Review by ahlukhouski2 — APPROVED** [review]

**Pull request merged**